src/NuGet.Core/NuGet.Packaging/NuGet.Packaging.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Description>NuGet's implementation for reading nupkg package and nuspec package specification files.</Description>
     <TargetFrameworks>net45;netstandard1.3</TargetFrameworks>
-    <NoWarn>$(NoWarn);CS1591;CS1574;CS1573</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1572</NoWarn>
```

Comment:
Looking at https://msdn.microsoft.com/en-us/library/hba91y19(v=vs.140).aspx. Why do we want to ignore these?